testnet/stacks-node/src/burnchains/bitcoin_regtest_controller.rs

```diff
@@ -1214,6 +1221,152 @@ impl BitcoinRegtestController {
         Some(tx)
     }
 
+    #[cfg(not(test))]
+    fn build_peg_out_request_tx(
+        &mut self,
+        _epoch_id: StacksEpochId,
+        _payload: PegOutRequestOp,
+        _signer: &mut BurnchainOpSigner,
+    ) -> Option<Transaction> {
+        unimplemented!()
+    }
+
+    #[cfg(test)]
+    fn build_peg_out_request_tx(
+        &mut self,
+        epoch_id: StacksEpochId,
+        payload: PegOutRequestOp,
+        signer: &mut BurnchainOpSigner,
+    ) -> Option<Transaction> {
+        let public_key = signer.get_public_key();
+        let max_tx_size = 230;
+        let dust_amount = 10000;
+
+        let output_amt = DUST_UTXO_LIMIT
+            + max_tx_size * self.config.burnchain.satoshis_per_byte
+            + payload.amount
```

Comment:
I see now that this is the _requested_ amount to be pegged out.  This should instead be whatever the going "dust" rate is.  Using `payload.amount` is a bug here.

Reply:
Ah I see. I'll update.

Reply:
Done :heavy_check_mark: